meta/documents/changelog_en.md

```diff
@@ -1,9 +1,15 @@
 # Release Notes for Elastic Export Belboon.de
 
+## v1.1.2 (2017-09-28)
+
+### Fixed
+- An issue was fixed which caused images loading problems.
```

Comment:
An issue was fixed which caused images not correctly to be exported.

Reply:
updated